tests/unit/OrgPerson.spec.ts

```diff
@@ -166,20 +166,21 @@ store.state.stateModel.tombstone.currentDate = '2020-03-30'
 describe('Org/Person component', () => {
   it('Loads the component and sets data for person', async () => {
     const wrapper: Wrapper<OrgPerson> = createComponent(validPersonData, -1, 0, null)
+    await Vue.nextTick()
     expect(wrapper.vm.$data.orgPerson).toStrictEqual(validPersonData)
-    expect(wrapper.vm.$data.isIncorporator).toBe(false)
-    expect(wrapper.vm.$data.isDirector).toBe(true)
-    expect(wrapper.vm.$data.isCompletingParty).toBe(true)
+    expect((wrapper.vm as any).isIncorporator).toBe(false)
```

Comment:
Are the type declarations required in the spec files?

Reply:
Do you mean, will this work if I don't cast to "any"? No.

I can't find a typecast that works here except "any". If you find one please let me know.

Reply:
No issue with 'any', just didn't know that the spec files were type checked is all.

Reply:
Yup, they're TS files after all.